src/components/organisms/pupil/OakQuizOrderQuestion/OakQuizOrder.tsx

```diff
@@ -0,0 +1,225 @@
+  Announcements,
+  DndContextProps,
+  MouseSensor,
+  TouchSensor,
+} from "@dnd-kit/core";
+import {
+  arrayMove,
+  SortableContext,
+  sortableKeyboardCoordinates,
+  useSortable,
+  verticalListSortingStrategy,
+} from "@dnd-kit/sortable";
+import { CSS } from "@dnd-kit/utilities";
+import { createPortal } from "react-dom";
+
+import { OakDraggable } from "@/components/molecules/OakDraggable";
+import { OakDroppable } from "@/components/molecules/OakDroppable";
+import { OakFlex } from "@/components/atoms";
+import { OakDragAndDropInstructions } from "@/components/molecules/OakDragAndDropInstructions";
+
+type OakQuizOrderItem = {
+  id: string;
+  label: string;
+};
+
+export type OakQuizOrderProps = {
+  initialItems: OakQuizOrderItem[];
+  onChange?: (items: OakQuizOrderItem[]) => void;
+};
+
+const ConnectedDraggable = memo(({ id, label }: OakQuizOrderItem) => {
+  const {
+    attributes,
+    listeners,
+    setNodeRef,
+    transform,
+    transition,
+    active,
+    over,
+  } = useSortable({ id });
+  const isGhostItem = active?.id === id;
+  const isGhostSlot = over?.id === id;
+  const style = {
+    transform: CSS.Transform.toString(transform),
+    transition,
+  };
+
+  return (
+    <OakDroppable isOver={isGhostSlot}>
+      <OakDraggable
+        ref={setNodeRef}
+        style={style}
+        isDisabled={isGhostItem}
+        {...attributes}
+        {...listeners}
+        aria-describedby={undefined}
+        aria-roledescription="sortable item"
+        aria-pressed={undefined}
+        aria-selected={!!attributes["aria-pressed"]}
+        role="option"
+      >
+        {label}
+      </OakDraggable>
+    </OakDroppable>
+  );
+});
+
+/**
+ * Facilitates DI for the DndContext
+ */
+export const injectDndContext = createContext<FC<DndContextProps>>(DndContext);
+
+/**
+ * A sortable list of items with drag and drop functionality. Items can be dragged over named slots to re-arrange them
+ *
+ * Keyboard navigation is supported with the `tab`, `space` and `arrow` keys
+ */
+export const OakQuizOrder = ({ initialItems, onChange }: OakQuizOrderProps) => {
+  const [items, setItems] = useState<OakQuizOrderItem[]>(initialItems);
+  const [activeId, setActiveId] = useState<string | null>(null);
+  const activeItem = items.find((item) => item.id === activeId);
+  const [prefersReducedMotion, setPrefersReducedMotion] = useState(false);
+  const sensors = useSensors(
+    useSensor(MouseSensor),
+    useSensor(TouchSensor),
+    useSensor(KeyboardSensor, {
+      coordinateGetter: sortableKeyboardCoordinates,
+      scrollBehavior: prefersReducedMotion ? "instant" : "smooth",
+    }),
+  );
+  const DndContext = useContext(injectDndContext);
+
+  /**
+   * Disable smooth scrolling during drag to ensure that the dragged item is always visible
+   */
+  useEffect(() => {
+    const originalScrollingBehaviour =
+      document.documentElement.style.scrollBehavior;
+    document.documentElement.style.scrollBehavior = "auto";
```

Comment:
This took me a while to work out — the reset styles in OWA apply `scrolling-behaviour: smooth` which is all well and good for normal scrolling but causes dragging an item below the fold to lag rendering it unusable. This switches that off for as long as the DnD UI is mounted.